lib/ruby_lsp/requests/support/rubocop_diagnostic.rb

```diff
@@ -61,11 +61,17 @@ def to_lsp_code_action
 
         sig { returns(LanguageServer::Protocol::Interface::Diagnostic) }
         def to_lsp_diagnostic
+          severity, message = if @offense.correctable?
+            [RUBOCOP_TO_LSP_SEVERITY[@offense.severity.name], @offense.message]
+          else
+            [LanguageServer::Protocol::Constant::DiagnosticSeverity::ERROR,
+             "This offense is not auto-correctable.\n#{@offense.message}",]
```

Comment:
Can we invert this an put the message first with two line breaks?
```suggestion
             "#{@offense.message}\n\nThis offense is not auto-correctable."]
```

Reply:
Yeah, that's what I mean with the two line breaks. It's easier to read when the violation message is split from this new note in my opinion.

Reply:
Thanks! I fixed it by adding a new line to the end of the message :)